🚀 Release datadog_flutter_plugin 1.0.0-beta.1 (#112)
DataDog/dd-sdk-flutter: packages/datadog_grpc_interceptor/README.md

# Datadog gRPC Interceptor / Plugin

> A plugin for use with the `DatadogSdk`, used to track performance of gRPC and enable Datadog Distributed Tracing.

> ⚠️ This plugin is still in Alpha / Developer Preview. 

## Getting started

> To utilize this plugin, create an instance of `DatadogGrpcInterceptor`, then pass it to your generated gRPC client:

Reviewer: ```suggestion
To use this plugin, create an instance of `DatadogGrpcInterceptor` and pass it to your generated gRPC client:
```

Author: I'll make this change as a separate pull request. The grpc interceptor isn't actually being released right now.
